wireless_modules/middle_module/main.py

```diff
@@ -0,0 +1,91 @@
+# Note: You can remove all the PRINT statements when everything is already tested
+
+import machine
+# from wireless_module import WirelessModule
```

Comment:
Why are these commented out?

Reply:
I was planning to use them later, I guess I should have removed it till I start using them again